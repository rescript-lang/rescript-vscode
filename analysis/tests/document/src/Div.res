let q = <div />
//        ^hov

// <div dangerous
//               ^com
