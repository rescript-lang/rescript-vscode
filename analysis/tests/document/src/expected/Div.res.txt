Hover document/src/Div.res 0:10
getLocItem #3: heuristic for <div>
{"contents": "```rescript\n(\n  string,\n  ~props: ReactDOMRe.domProps=?,\n  array<React.element>,\n) => React.element\n```"}

Complete document/src/Div.res 3:17
posCursor:[3:17] posNoWhite:[3:16] Found expr:[3:4->3:17]
JSX <div:[3:4->3:7] dangerous[3:8->3:17]=...[3:8->3:17]> _children:None
Completable: Cjsx([div], dangerous, [dangerous])
[{
    "label": "dangerouslySetInnerHTML",
    "kind": 4,
    "tags": [],
    "detail": "{\"__html\": string}",
    "documentation": null
  }]

